static/public/index.html

```diff
@@ -4443,6 +4448,3396 @@ <h3 class="text-subtitle font-bold text-dark">Pickly</h3>
         </nav>
       </div>
     </section>
+
+    <section class="bg-grey-bg">
+      <div class="atom molecules organisms">
+        <h2 class="header">Post Edit</h2>
+        <div class="parent">
+          <div class="title-header">
+            <h3 class="title text-lg">General</h3>
+          </div>
+          <div class="child flex-col">
+            <div
+              class="misc-box bg-white shadow-soft rounded-md flex items-center p-m mb-10"
+            >
+              <img
+                src="./img/large-avatar.png"
+                alt="filled avatar"
+                class="w-10 mr-m"
+              />
+              <input
+                class="border border-grey-shd5 py-xsvv px-m text-sm font-normal text-grey-shd1 w-64 md:w-34xl rounded-md focus:outline-none"
+                type="text"
+                placeholder="What do you want to ask about?"
+                class=""
+              />
+            </div>
+          </div>
+        </div>
+
+        <div class="parent">
+          <div class="title-header">
+            <h3 class="title text-lg">Image Poll - Default</h3>
```

Comment:
1. Change weight for font in upload image and group tap
2. When i change choice in tab group , style don't change 
3. When i choose any checkbox , happen slowly animation effect
4. Change font weight for selectbox is not identical as design